### PR TITLE
chore: add the terraform-example-java-dynamic-point-of-sale repo

### PR DESCRIPTION
**Pull request by Shabirmean**
### Description:
- Onboard the [terraform-example-java-dynamic-point-of-sale](https://github.com/GoogleCloudPlatform/terraform-example-java-dynamic-point-of-sale) repository to the CFT CI
- This is the repo hosting the solution for the user pattern 2.2 of project v10r

### CODE REVIEW

File: infra/terraform/test-org/org/locals.tf [hunk 1]
```diff
@@ -115,6 +115,13 @@ locals {
       description = "Deploys a web-based ecommerce app into a multi-cluster Google Kubernetes Engine setup."
       owners      = ["NimJay"]
     },
+    {
+      name        = "terraform-example-java-dynamic-point-of-sale"
+      short_name  = "java-dynamic-point-of-sale"
+      org         = "GoogleCloudPlatform"
+      description = "Deploys a dynamic Java webapp into a Google Kubernetes Engine cluster."
+      owners      = ["@GoogleCloudPlatform/dee-platform-ops"]
```
Comment (bharathkkb) [inline infra/terraform/test-org/org/locals.tf:123]: We don't support groups just yet https://github.com/GoogleCloudPlatform/cloud-foundation-toolkit/issues/1505. Could you switch to users for now?

Comment (Shabirmean) [reply infra/terraform/test-org/org/locals.tf]: Done!

**Review by bharathkkb — APPROVED** [review]

**Comment by Shabirmean** [comment]
Thank you for the review @bharathkkb 
Please merge upon green 🙏🏼

**Comment by apeabody** [comment]
@bharathkkb - I can apply/merge this with the others.

**Comment by apeabody** [comment]
applied

**Pull request merged**